packages/jest-preset/jest-preset.js

```diff
@@ -17,6 +17,8 @@ if (Number(jestMajorVersion) < 26) {
   );
 }
 
+const enableMocks = process.env.ENABLE_MOCKS === 'true';
```

Comment:
I would like to have a better name here as `mocks` is a general term and it's not obvious that we are talking about service worker code that mocks specific network reqeusts.

Reply:
We took that name from crate, where it is also `ENABLE_MOCKS`.

Reply:
I know, but is it required that it has the same name? I'm happy to keep the name if you think it makes sense.
But to me, it isn't very descriptive of what it is doing though. And I would also try to prevent leaking internal company "requirements" into Hops - whenever possible.

Wow about `ENABLE_MSW` or `ENABLE_SW_MOCKS`? Not sure how expressive `SW` and `MSW` are though.

Reply:
We'll rename it to `ENABLE_MSW` to make it more dedicated to the `msw` thingie. But the naming should suffice then. If users write that env var, they likely have informed about its existence in beforehand.

Reply:
@jhiode done; ptal again 🙏